src/assets/ethereum-loading.svg

```diff
@@ -0,0 +1,23 @@
+<!--?xml version="1.0" encoding="UTF-8"?-->
```

Comment:
If we use this via an `<img>`, we should optimize this through [svgo](https://jakearchibald.github.io/svgomg/)